packages/utils/src/lib/audio.ts

```diff
@@ -1,22 +1,31 @@
+import {
+  AudioState,
+  resetAudioState,
+  setAudio,
+  setAudioProgress,
+  setAudioUrl,
+  setIsAudioPlaying,
+} from "@newm-web/modules";
 import { Howl } from "howler";
-import { useCallback, useEffect, useMemo, useState } from "react";
+import { useCallback, useEffect, useMemo } from "react";
+import { useDispatch, useSelector } from "react-redux";
 
 /**
  * Hook used to play an audio file using the Howl library.
  */
 export const usePlayAudioUrl = () => {
-  const [audio, setAudio] = useState<Howl>();
-  const [audioUrl, setAudioUrl] = useState<string>();
-  const [isAudioPlaying, setIsAudioPlaying] = useState<boolean>(false);
-  const [audioProgress, setAudioProgress] = useState<number>(0);
+  const dispatch = useDispatch();
+  const { audio, audioUrl, audioProgress, isAudioPlaying }: AudioState =
+    useSelector((state: { audio: AudioState }): AudioState => state.audio);
```

Comment:
In our main apps we use a custom `useAppSelector` hook but without `store.ts` and access to the `RootState` I wasn't able to integrate that here. Is there a better way to do this? Move over `store.ts` and other redux tooling to the packages directory?